content/blog/2023-03-29-ethereum-classic-ossification-and-social-scalability/index.md

```diff
@@ -0,0 +1,105 @@
+
+This problem is usually called “attack surface”, which means that the larger the surface of the system is, the easier it will be to poke holes in it. In other words, the more code and rules it has, the easier it will be to find ways to attack it.
+
+This is why the term “ossification” is usually used and is a goal in the industry. Ossification means to reach a point where the network is sufficiently functional to warrant a complete halt in its upgrades and bug fixes. Then, all new features and functionality may continue to be built on top in higher layers.
+
+In the case of networks as Ethereum Classic, the attack surface problem has three angles: Complexity, politics, and change.
+
+## Complexity
+
+The complexity attack surface: The more complex rules and code needed to build the system, the more hidden ways hackers will find to attack it.
+
+Complexity also raises the risk of hidden bugs or unforeseen failures.
+
+The way of reducing this surface is by reducing complexity or reaching a point of ossification where the system is functional enough to support all new features and innovation in external complementary protocols.
+
+Examples of this may be rollups, channels, and other scaling systems built on top of ETC, instead of sharding it internally or integrating them into the network.
+
+## Protocol Politics
+
+The argument attack surface: The more decisions to argue over, the more the opportunity to convince the ecosystem to accept bad upgrades that reduce decentralization.
+
+In blockchain networks there are several constituents that participate in the debates when there are new proposals for change. These debates may become very passionate and even toxic at times. Sometimes, parties with enough budgets, charisma, or influence may actually convince participants to make horrible changes. The most famous of these examples is the migration of Ethereum from proof of work to proof of stake, a totally prone to centralization system.
+
+These processes turn developers, economic node operators, and miners into political animals fighting for their self interest.
+
+Ossification would avoid this problem entirely as there would be no possibility of tricking the ecosystem with fancy arguments and gimmicks anymore.
+
+## Change
+
+The change attack surface: The more changes and upgrades there are, the more opportunities to introduce faulty rules or bugs.
+
+Similar to protocol politics, just the frequency of change, even if necessary and benevolent, is a risk to the system. 
+
+It has happened in the past, for example, that Bitcoin had a bug nobody had seen, and when it was discovered they had to introduce an urgent change.
+
+Save hidden bugs that may still be there, ossification would eliminate this risk, at least of introducing new bugs and errors, because there would be no introduction of new code or rules to the protocol.
```

Comment:
Line 93: Reword awkward structure
```suggestion
Except for those hidden and already present in the code, ossification eliminates the introduction of new bugs or errors because there would be no new code or rules deployed to the protocol.
```